spark/src/main/scala/io/spark/ddf/etl/TransformationHandler.scala

```diff
@@ -96,26 +101,20 @@ class TransformationHandler(mDDF: DDF) extends CoreTransformationHandler(mDDF) {
     val rMapped = dfrdd.map {
```

Comment:
After evaluating R exp with Rsession, rMapped became another type of variable:

[error]  found   : org.apache.spark.rdd.RDD[Nothing]
[error]  required: org.apache.spark.rdd.RDD[org.rosuda.REngine.REXP]

The error point to line 129 of this file(I can't comment on that line)